war/src/main/java/io/lumeer/engine/controller/OrganizationFacade.java

```diff
@@ -0,0 +1,463 @@
+      DataDocument document = dataStorage.readDocumentIncludeAttrs(LumeerConst.Organization.COLLECTION_NAME, organizationIdFilter(organizationId), Collections.singletonList(LumeerConst.Organization.ATTR_ORG_NAME));
+      return document != null ? document.getString(LumeerConst.Organization.ATTR_ORG_NAME) : null;
+   }
+
+   /**
+    * Reads the specific organization metadata.
+    *
+    * @param organizationId
+    *       id of the organization
+    * @param metaName
+    *       name of the meta attribute
+    * @return meta attribute value
+    */
+   public String readOrganizationMetadata(final String organizationId, final String metaName) {
+      DataDocument document = dataStorage.readDocumentIncludeAttrs(LumeerConst.Organization.COLLECTION_NAME, organizationIdFilter(organizationId), Collections.singletonList(metaName));
+      return document != null ? document.getString(metaName) : null;
+   }
+
+   /**
+    * Updates single organization metadata.
+    *
+    * @param organizationId
+    *       id of the organization
+    * @param metaName
+    *       name of the meta attribute to update
+    * @param value
+    *       meta attribute value
+    */
+   public void updateOrganizationMetadata(final String organizationId, final String metaName, final String value) {
+      updateOrganizationMetadata(organizationId, new DataDocument(metaName, value));
+   }
+
+   /**
+    * Updates multiple organization metadata.
+    *
+    * @param organizationId
+    *       id of the organization
+    * @param meta
+    *       key-value pairs of metadata to update
+    */
+   public void updateOrganizationMetadata(final String organizationId, final DataDocument meta) {
+      dataStorage.updateDocument(LumeerConst.Organization.COLLECTION_NAME, meta, organizationIdFilter(organizationId));
+   }
+
+   /**
+    * Removes the specific organization metadata.
+    *
+    * @param organizationId
+    *       id of the organization
+    * @param metaName
+    *       name of the meta attribute to remove
+    */
+   public void dropOrganizationMetadata(final String organizationId, final String metaName) {
+      dataStorage.dropAttribute(LumeerConst.Organization.COLLECTION_NAME, organizationIdFilter(organizationId), metaName);
+   }
+
+   /**
+    * Creates new organization in the system database.
+    *
+    * @param organizationId
+    *       id of the new organization to create
+    * @param organizationName
+    *       name of the new organization to create
+    */
+   public void createOrganization(final String organizationId, final String organizationName) {
+      DataDocument document = new DataDocument(LumeerConst.Organization.ATTR_ORG_ID, organizationId)
+            .append(LumeerConst.Organization.ATTR_ORG_NAME, organizationName)
+            .append(LumeerConst.Organization.ATTR_ORG_DATA, new DataDocument())
+            .append(LumeerConst.Organization.ATTR_USERS, new DataDocument());
```

Comment:
@kubedo8 Yes, that's corret. But you need to append at least the first `ATTR_ORG_NAME` attribute because of `organizationName`.